src/engine/NodeHelpers.cs

```diff
@@ -115,4 +115,27 @@ public static void ReParent(this Node node, Node newParent)
         node.GetParent().RemoveChild(node);
         newParent.AddChild(node);
     }
+
+    /// <summary>
+    ///   Get the material of this scene's model.
+    /// </summary>
+    /// <param name="node">Node to get material from.</param>
+    /// <param name="modelPath">Path to model within the scene. If null takes scene root as model.</param>
+    /// <returns>ShaderMaterial of the GeometryInstance.</returns>
+    public static ShaderMaterial GetMaterial(this Node node, string modelPath = null)
```

Comment:
I'd use godot's `NodePath` datatype here, as well as in `SceneDisplayer.GetMaterial`